dist/smartbanner.js

```diff
@@ -421,14 +421,18 @@ var SmartBanner = function () {
       var bannerDiv = document.createElement('div');
       document.querySelector('body').appendChild(bannerDiv);
       bannerDiv.outerHTML = this.html;
-      setContentPosition(this.height);
+      if (this.options.disablePositioning !== 'true') {
```

Comment:
Looking good. Let's implement a boolean getter for this to DRY up, i.e. `positioningDisabled`, as we have the `platformEnabled`.